website/src/app/[lang]/[region]/(website)/open-source/(sections)/contributors.tsx

```
@@ -17,10 +40,50 @@ export async function OpenSourceContributors({ lang }: DefaultParams) {
 	const heading = metadata.heading;
 
 	const contributors = await getContributors();
+	const { totalCommitsData } = await getCommits();
 	const totalContributors = contributors.length;
 
+	// Collect the latest commit date for each contributor
+	const latestCommitDates = new Map<number, Date>();
+	totalCommitsData.forEach((commit: GitHubCommit) => {
+		if (commit.author && commit.author.id && commit.commit.author.date) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)